custom_components/balboa/__init__.py

```diff
@@ -1,23 +1,24 @@
-"""The Balboa Spa integration."""
+"""The Balboa Spa Client integration."""
 import asyncio
 import logging
 import time
 from typing import Any, Dict
 
-from pybalboa import BalboaSpaWifi
+import homeassistant.helpers.config_validation as cv
+import homeassistant.util.dt as dt_util
 import voluptuous as vol
-
 from homeassistant.config_entries import SOURCE_IMPORT, ConfigEntry
 from homeassistant.const import CONF_HOST, CONF_NAME
 from homeassistant.core import HomeAssistant, callback
-import homeassistant.helpers.config_validation as cv
-from homeassistant.helpers.dispatcher import (
-    async_dispatcher_connect,
-    async_dispatcher_send,
-)
+from homeassistant.exceptions import ConfigEntryNotReady
+from homeassistant.helpers.device_registry import CONNECTION_NETWORK_MAC
+from homeassistant.helpers.dispatcher import (async_dispatcher_connect,
+                                              async_dispatcher_send)
```

Comment:
For import sections, please use the original formatting here.  The integration is written so maybe one day I will try to put it into the base again, and they require the style of formatting I had previously.